basic-conditionals.tex

```diff
@@ -0,0 +1,400 @@
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja od druge.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZlt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je manjsa ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{if} \PY{p}{(}\PY{n}{a} \PY{o}{\PYZgt{}}\PY{o}{=} \PY{n}{b}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Prva stevilka je vecja ali enaka drugi.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+12 12
+\tcblower
+Stevilki sta enaki.\\
+Prva stevilka je manjsa ali enaka drugi.\\
+Prva stevilka je vecja ali enaka drugi.
+\end{inout}
+
+\begin{inout}
+3 7
+\tcblower
+Stevilki sta razlicni.\\
+Prva stevilka je manjsa od druge.\\
+Prva stevilka je majnsa ali enaka drugi.
+\end{inout}
+
+\end{examples}
+
+
+\begin{errors}
+  Pri primerjavi moramo uporabiti dva enačaja (\verb+==+). Če uporabimo samo en enačaj, kot v matematiki (torej \verb+=+), se bo program sicer zagnal, vendar ne bo deloval pravilno.
+  Enojni enačaj uporabljamo samo pri nastavljanju vrednosti neki spremenljivki, in nikoli v običajnih oklepajih v \verb+if+ stavku.
+\end{errors}
+
+\begin{errors}
+  Taka primerjava deluje samo na številkah. Če želimo primerjati med dvema besediloma, za to potrebujemo posebno funkcijo \verb+strcmp+, ki jo bomo podrobneje spoznali, ko bomo govorili o besedilu.
+\end{errors}
+
+\begin{examples}
+
+Primer uporabe stavka \verb+else+.
+
+\begin{Verbatim}[commandchars=\\\{\}]
+\PY{c+cp}{\PYZsh{}}\PY{c+cp}{include} \PY{c+cpf}{\PYZlt{}stdio.h\PYZgt{}}
+
+\PY{k+kt}{int} \PY{n+nf}{main}\PY{p}{(}\PY{p}{)} \PY{p}{\PYZob{}}
+    \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{Vpisi PIN:}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{k+kt}{int} \PY{n}{pin}\PY{p}{;}
+    \PY{n}{scanf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{\PYZpc{}d}\PY{l+s}{\PYZdq{}}\PY{p}{,} \PY{o}{\PYZam{}}\PY{n}{pin}\PY{p}{)}\PY{p}{;}
+    \PY{k}{if} \PY{p}{(}\PY{n}{pin} \PY{o}{=}\PY{o}{=} \PY{l+m+mi}{42}\PY{p}{)} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{PIN je pravilen.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}} \PY{k}{else} \PY{p}{\PYZob{}}
+        \PY{n}{printf}\PY{p}{(}\PY{l+s}{\PYZdq{}}\PY{l+s}{PIN je napacen.}\PY{l+s+se}{\PYZbs{}n}\PY{l+s}{\PYZdq{}}\PY{p}{)}\PY{p}{;}
+    \PY{p}{\PYZcb{}}
+    \PY{k}{return} \PY{l+m+mi}{0}\PY{p}{;}
+\PY{p}{\PYZcb{}}
+\end{Verbatim}
+
+\begin{inout}
+Vpisi PIN: \\
```

Comment:
Kot receno, bi mi bilo ljubse, ce ne bi imeli takih interaktivnih stvari. Ampak ce vztrajas lahko tudi pustimo tule. Ampak potem bi bilo fino, da je slogovno uniformno z nacinom, ki ga je uporabila Ella.

Reply:
Pomoje je smiselno imeti interaktivne primere. Res je, da ne bodo nikoli tako programirali na putki, ali kje drugje, ampak razumevanje kode je veliko lažje, če piše 

```c++
printf("Napisi svojo starost:\n");
scanf("%d", &starost);
printf("Cez 10 let bos star/a %d.\n", starost+10);
```
kot 
```c++
scanf("%d", &starost);
printf("Cez 10 let bos star/a %d.\n", starost+10);
```

Nekako glavna razlika tukaj je, da program s kodo (in v slovenščini) vnaprej napove, kaj bo naredil v prihodnje, namesto da moramo dvakrat prebrati in ugotavljati za nazaj, kaj naj bi bil vhod, ali pa da mroamo imeti odstavek spremnega besedila samo zato, da lahko razumemo 10 vrstic kode.